lib/cc/analyzer/container.rb

```diff
@@ -116,7 +154,11 @@ def reap_running_container
       end
 
       def timeout
-        (ENV["CONTAINER_TIMEOUT_SECONDS"] || DEFAULT_TIMEOUT).to_i
+        ENV.fetch("CONTAINER_TIMEOUT_SECONDS", DEFAULT_TIMEOUT).to_i
```

Comment:
One important behavioral difference here is that if `ENV["CONTAINER_TIMEOUT_SECONDS"]` exists but is nil, this will be 0